test/logger-usage/src/main/java/org/elasticsearch/test/loggerusage/ESLoggerUsageChecker.java

```diff
@@ -59,6 +57,7 @@
     public static final List<String> LOGGER_METHODS = Arrays.asList("trace", "debug", "info", "warn", "error");
     public static final String IGNORE_CHECKS_ANNOTATION = "org.elasticsearch.common.SuppressLoggerChecks";
 
+    @SuppressForbidden(reason = "command line tool")
```

Comment:
can you use the `@ SuppressForbidden` from lucene here? I think the test framework has dependencies on lucene no?